styles/index.scss

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 @import "typography";
 @import "images";
 @import "code";
+@import "tables";
```

Comment:
File name should probably be `table_wrapper` to match the plugin, and should be in the plugins section below?

Maybe we should keep the inner tables styling in the `tables` file, so that anyone can benefit from those base styles, and then just the wrapper styling in the `table_wrapper` file?